From d2e8e6b1f6a6121e57b85f396ced4f59c861a837 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Mon, 19 Aug 2013 14:08:56 +0800
Subject: [PATCH 238/310] ENGR00275632 Remove unused fstab file for imx6 soc

Each board has its own fstab in its board directory.

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 imx6/etc/fstab.freescale      | 12 ------------
 imx6/etc/fstab_nand.freescale | 10 ----------
 2 files changed, 22 deletions(-)
 delete mode 100644 imx6/etc/fstab.freescale
 delete mode 100644 imx6/etc/fstab_nand.freescale

diff --git a/imx6/etc/fstab.freescale b/imx6/etc/fstab.freescale
deleted file mode 100644
index e5001be..0000000
--- a/imx6/etc/fstab.freescale
+++ /dev/null
@@ -1,12 +0,0 @@
-
-# Android fstab file.
-#<src>                                                  <mnt_point>         <type>    <mnt_flags and options>                       <fs_mgr_flags>
-# The filesystem that contains the filesystem checker binary (typically /system) cannot
-# specify MF_CHECK, and must come before any filesystems that do specify MF_CHECK
-/devices/platform/sdhci-esdhc-imx.2/mmc_host/mmc1 /mnt/extsd vfat defaults voldmanaged=sdcard:auto
-/devices/platform/fsl-ehci /mnt/udisk vfat defaults voldmanaged=sdcard:auto
-/dev/block/mmcblk0p5    /system  ext4    ro                                                                               wait
-/dev/block/mmcblk0p4    /data    ext4    nosuid,nodev,nodiratime,noatime,nomblk_io_submit,noauto_da_alloc,errors=panic    wait,encryptable=footer
-/dev/block/mmcblk0p6    /cache   ext4    nosuid,nodev,nomblk_io_submit				                          wait
-/dev/block/mmcblk0p7    /device  ext4    ro,nosuid,nodev				                                  wait
-
diff --git a/imx6/etc/fstab_nand.freescale b/imx6/etc/fstab_nand.freescale
deleted file mode 100644
index 55e65f6..0000000
--- a/imx6/etc/fstab_nand.freescale
+++ /dev/null
@@ -1,10 +0,0 @@
-# Android fstab file.
-#<src>                                                  <mnt_point>         <type>    <mnt_flags>                                                                         <fs_mgr_flags>
-# The filesystem that contains the filesystem checker binary (typically /system) cannot
-# specify MF_CHECK, and must come before any filesystems that do specify MF_CHECK
-
-# below is for ubi support, uncomment to support ubifs(NAND)
-ubi0:system	/system		ubifs 	ro					wait
-ubi0:data       /data           ubifs   nosuid,nodev,nodiratime,noatime         wait
-ubi0:cache      /cache          ubifs   nosuid,nodev                            wait
-ubi0:device     /device         ubifs   nosuid,nodev                         wait
-- 
1.8.0

